framework/tlvf/yaml/tlvf/WSC/WSC_Attributes.yaml

```diff
@@ -264,6 +264,33 @@ sWscAttrVersion2:
     _type: uint8_t
     _value: 0x20
 
+sWscAttrVendorExtMultiAp:
+  _type: struct
+  attribute_type:
+    _type: eWscAttributes
+    _value: ATTR_VENDOR_EXTENSION
+  data_length:
+    _type: uint16_t
+    _value: WSC_VENDOR_EXTENSIONS_LENGTH
+  vendor_id_0:
+    _type: uint8_t
+    _value: WSC_VENDOR_ID_WFA_1
+  vendor_id_1:
+    _type: uint8_t
+    _value: WSC_VENDOR_ID_WFA_2
+  vendor_id_2:
+    _type: uint8_t
+    _value: WSC_VENDOR_ID_WFA_3
+  subelement_id:
+    _type: uint8_t
+    _value: 0x6
+  subelement_length:
+    _type: uint8_t
+    _value: 0x1
+  subelement_value:
+    _type: uint8_t
+    _value: TEARDOWN
```

Comment:
Why here you used enum and on sWscAttrVersion2 you used value?

Reply:
> Why here you used enum and on sWscAttrVersion2 you used value?

This is copied from how it was before this PR, that's why it uses an enum.